pkg/devices/lock.go

```diff
@@ -0,0 +1,67 @@
+package devices
+
+import (
+	"fmt"
+
+	"github.com/vapor-ware/synse-sdk/sdk"
+)
+
+const (
+	stateLock        = "lock"
+	stateUnlock      = "unlock"
+	statePulseUnlock = "pulseUnlock"
+)
+
+// lstate holds the lock state.
+var lstate string
+
+// Lock is the handler for the emulated Lock device(s).
+var Lock = sdk.DeviceHandler{
+	Name:  "lock",
+	Read:  lockRead,
+	Write: lockWrite,
+}
+
+// lockRead is the read handler for the emulated Lock device(s). It
+// returns the state values for the device.
+func lockRead(device *sdk.Device) ([]*sdk.Reading, error) {
+	if lstate == "" {
+		lstate = stateLock
+	}
+
+	stateReading, err := device.GetOutput("lock.state").MakeReading(lstate)
+	if err != nil {
+		return nil, err
+	}
+
+	return []*sdk.Reading{
+		stateReading,
```

Comment:
for the returned state, we should use a subset of the statuses that we are using in the i2c device: https://github.com/vapor-ware/synse-plugins-internal/blob/master/i2c/devices/rci3525/device.go#L73

In this case, we wouldn't have to worry about mechanical unlock, so the two states we should return are `locked` or `unlocked_electrically`. This isn't a huge deal, since its well known that readings can vary between devices even if they are the same type, but having some consistency here would be useful for future testing